### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by hagabbar** [hagabbar/GenNet#3]
- Add in constraints to loss function that effectively train the network to assume noise has zero mean and with a user-defined standard deviation.
- Does this make sense? 
- Basically, you should research more into how to directly edit the loss function either in Keras or TensorFlow.
